code/modules/projectiles/boxes_magazines/external/rifle.dm

```diff
@@ -21,3 +21,60 @@
 /obj/item/ammo_box/magazine/m223/phasic
 	name = "toploader magazine (.223 Phasic)"
 	ammo_type = /obj/item/ammo_casing/a223/phasic
+
+// .38 (Battle Rifle) //
+
+/obj/item/ammo_box/magazine/m38
+	name = "\improper battle rifle magazine (.38)"
```

Comment:
Oops, this isn't a proper noun. `\improper` not necessary

Reply:
fug